src/swarm/connect.js

```diff
@@ -20,11 +24,7 @@ module.exports = (createCommon, options) => {
 
     after(() => common.teardown())
 
-    it('should connect to a peer', (done) => {
-      ipfsA.swarm.connect(ipfsB.peerId.addresses[0], done)
-    })
-
-    it('should connect to a peer (promised)', () => {
+    it('should connect to a peer', () => {
       return ipfsA.swarm.connect(ipfsB.peerId.addresses[0])
```

Comment:
Should probably assert that the remote peer is now in the peer list (and wasn't to start with).